src/api-service/__app__/onefuzzlib/extension.py

```diff
@@ -150,7 +150,7 @@ def azmon_extension(
         "type": "AzureMonitorLinuxAgent",
         "typeHandlerVersion": "1.9",
         "autoUpgradeMinorVersion": False,
```

Comment:
In this extension we have `autoUpgradeMinorVersion` set to False, but on line 175, we set that extension autUpgradeMinorVersion to True. Do we think that AzureMonitorLinuxAgent could reboot on upgrade and AzureSecurityLinuxAgent wouldn't?

Reply:
AzureMonitorLinuxAgent was set to False because a previous upgrade caused issues based off agent naming conventions. The current version is a valid and supported version. 

AzureSecurityLinuxAgent doesn't have this issue. 

If the agent restarted there would be zero problems. Both agents support restarts and can re-establish their connections with the paired agent. I do it all the time manually on the nodes.